package.json

```diff
@@ -109,7 +109,8 @@
     "Sebastian Poxhofer <sebastian@poxhofer.at>",
     "Henry Sachs <henrysachs@gmail.com>",
     "Arkadiusz Kosmala <kosmala.arkadiusz@gmail.com>",
-    "Markus Siebert <mail@markussiebert.com>"
+    "Markus Siebert <mail@markussiebert.com>",
+    "Sergey Vedmak <serg.vedmak@gmail.com>"
```

Comment:
@sergVedmak Please review if this is how you'd like to be credited for your work. Feel free to suggest changes. 😉

Reply:
@HonkingGoose looks good 🙂  thank you!